cinema/models.py

```diff
@@ -5,6 +5,23 @@ class Movie(models.Model):
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
+    genres = models.ManyToManyField(to="Genre", related_name="movies")
+    actors = models.ManyToManyField(to="Actor", related_name="movies")
 
     def __str__(self):
         return self.title
+
+
+class Actor(models.Model):
+    first_name = models.CharField(max_length=63)
+    last_name = models.CharField(max_length=63)
+
```

Comment:
It is a good practice to always override str method. Implement it everywhere

Reply:
Thanks for review, but I think cinema_data should be presented in the repository :)